GVFS/GVFS.Upgrader/UpgradeOrchestrator.cs

```diff
@@ -114,8 +119,16 @@ public void Execute()
 
                 if (this.ExitCode == ReturnCode.GenericError)
                 {
-                    error = Environment.NewLine + "ERROR: " + error;
-                    this.output.WriteLine(error);
+                    StringBuilder sb = new StringBuilder();
+                    sb.AppendLine();
+                    sb.Append("ERROR: " + error);
+
+                    sb.AppendLine();
+                    sb.AppendLine();
+
+                    sb.AppendLine($"Installation logs can be found at: {this.logDirectory} with the installation id: {this.installationId}");

```

Comment:
Why can't this be the file name that information will be saved to?  Will the user even know what an "installation id" is and what is it used for and that they need to look for a file with that in the name?

Reply:
The upgrader can generate multiple logs - in particular - each installer that is run can generate there own set of set of logs. My thought was the installation ID will be obvious enough that when a user goes and looks at this directory, they will be able to find the relevent files. Maybe we could just direct the user to the single top level log file, but then that doesn't include any hints about the other possible installer logs. 

The previous approach was to write all logs to a new directory for each upgrade. This had the advantage of pointing the user to a single directory with relevent logs, but the downside of the log related logic not being set up to deal with sub directories.

I think the current approach is a good compromise - but we can tweak this as needed.

Reply:
I think Kevin brings up a good point that users might not know what an "installation id" is.

If we want to stick with this approach I think the experience could be improved by a small change to the message.  Rather than:

> with the installation id: {this.installationId}

If we said something like:

> with file names that end with {this.installationId}

Thoughts?

Alternatively, if you wanted to keep the directory structure perhaps `gvfs log` could search all files in the logs directories (including those in subfolders) and return the latest?

Reply:
> with file names that end with installation id {this.installationId}

I prefer this approach for now (or a similar wordsmithing) rather than tweaking the directory structure of how we write out logs for now. `installation id` is kind of an arbitrary name. I could also do:

`file names with the suffix {this.installationId}`?